### PR TITLE
IGVF-929-update-configs

### PR DESCRIPTION
**Pull request by khzlin**

### CODE REVIEW

File: src/igvfd/searches/configs/prediction.py [hunk 1]
```diff
@@ -16,10 +16,42 @@ def prediction():
             'lab.title': {
                 'title': 'Lab'
             },
+            'prediction_type': {
```
Comment (iyoungworth) [inline src/igvfd/searches/configs/prediction.py:19]: prediction_type duplicated here

Comment (iyoungworth) [reply src/igvfd/searches/configs/prediction.py:19]: Can you also remove trailing comma L76, and empty L78

File: src/igvfd/searches/configs/primary_cell.py [hunk 2]
```diff
@@ -42,6 +42,9 @@ def primary_cell():
             },
```
Comment (iyoungworth) [inline src/igvfd/searches/configs/primary_cell.py:42]: L19: donors.taxa should be removed (have 'taxa' listed already)

File: src/igvfd/searches/configs/primary_cell.py [hunk 3]
```diff
@@ -42,6 +42,9 @@ def primary_cell():
...
             'virtual': {
                 'title': 'Virtual'
+            },
+            'biomarkers.classification': {
+                'titlle': 'Biomarkers Classification'
```
Comment (iyoungworth) [inline src/igvfd/searches/configs/primary_cell.py:47]: "titlle" typo

File: src/igvfd/searches/configs/publication.py [hunk 4]
```diff
@@ -69,8 +69,8 @@ def publication():
             'status': {
                 'title': 'Status'
             },
-            'identifiers': {
-                'title': 'Identifiers'
+            'publication_identifiers': {
+                'title': 'Publication Identifiers'
             },
```
Comment (iyoungworth) [inline src/igvfd/searches/configs/publication.py:74]: trailing comma

File: src/igvfd/searches/configs/tissue.py [hunk 5]
```diff
@@ -39,17 +39,22 @@ def tissue():
             },
             'virtual': {
                 'title': 'Virtual'
+            },
+            'biomarkers.classification': {
+                'titlle': 'Biomarkers Classification'
```
Comment (iyoungworth) [inline src/igvfd/searches/configs/tissue.py:44]: "titlle" typo

File: src/igvfd/searches/configs/whole_organism.py [hunk 6]
```diff
@@ -51,6 +51,7 @@ def whole_organism():
                     'biomarkers.classification',
                     'biosample_term.term_name',
                     'disease_terms.term_name',
+                    'virtual',
```
Comment (iyoungworth) [inline src/igvfd/searches/configs/whole_organism.py:54]: L29, can you updated title to "Disease Terms" for consistency?

Comment (iyoungworth) [reply src/igvfd/searches/configs/whole_organism.py:54]: L100, please remove submitted_by

**Review by iyoungworth — CHANGES_REQUESTED** [review]
Some small typos and edits requested

File: src/igvfd/searches/configs/alignment_file.py [hunk 7]
```diff
@@ -25,5 +72,5 @@ def alignment_file():
             'status': {
                 'title': 'Status'
             }
-        }
+        },
```
[the remark this answers is not in the document]
Comment (khzlin) [reply src/igvfd/searches/configs/alignment_file.py:75]: fixed

File: src/igvfd/searches/configs/auxiliary_set.py [hunk 8]
```diff
@@ -18,8 +18,43 @@ def auxiliary_set():
...
                 'title': 'Auxiliary Type'
-            }
+            },
+            'donors.taxa': {
+                'title': 'Taxa'
+            },
+            'collections': {
+                'title': 'Collections',
+            },
+            'lab.title': {
```
[the remark this answers is not in the document]
Comment (khzlin) [reply src/igvfd/searches/configs/auxiliary_set.py:28]: fixed

File: src/igvfd/searches/configs/construct_library.py [hunk 9]
```diff
@@ -21,8 +18,42 @@ def cell_line():
...
             'selection_criteria': {
                 'title': 'Selection Criteria'
+            },
+            'collections': {
+                'title': 'Collections',
+            },
+            'lab.title': {
+                'title': 'Lab'
+            },
+            'award.component': {
```
[the remark this answers is not in the document]
Comment (khzlin) [reply src/igvfd/searches/configs/construct_library.py:28]: fixed

File: src/igvfd/searches/configs/curated_set.py [hunk 10]
```diff
@@ -25,6 +22,18 @@ def curated_set():
             'curated_set_type': {
                 'title': 'Curated Set Type'
             },
+            'collections': {
+                'title': 'Collections',
+            },
+            'lab.title': {
```
[the remark this answers is not in the document]
Comment (khzlin) [reply src/igvfd/searches/configs/curated_set.py:28]: fixed

File: src/igvfd/searches/configs/multiplexed_sample.py [hunk 11]
```diff
@@ -21,8 +30,37 @@ def multiplexed_sample():
             },
             'status': {
                 'title': 'Status'
+            },
+            'biomarkers.classification': {
+                'titlle': 'Biomarkers Classification'
```
[the remark this answers is not in the document]
Comment (khzlin) [reply src/igvfd/searches/configs/multiplexed_sample.py:35]: fixed

File: src/igvfd/searches/configs/multiplexed_sample.py [hunk 12]
```diff
@@ -21,8 +30,37 @@ def multiplexed_sample():
...
+                    'biomarkers.classification',
+                ]
+            },
+            {
+                'title': 'Provenance',
+                'facet_fields': [
+                    'collections',
+                    'lab.title',
+                    'award.component',
+                    'source.title',
```
[the remark this answers is not in the document]
Comment (khzlin) [reply src/igvfd/searches/configs/multiplexed_sample.py:54]: fixed

File: src/igvfd/searches/configs/prediction.py (hunk 1 above)
Comment (khzlin) [reply src/igvfd/searches/configs/prediction.py:19]: fixed

File: src/igvfd/searches/configs/primary_cell.py (hunk 3 above)
Comment (khzlin) [reply src/igvfd/searches/configs/primary_cell.py:47]: fixed

File: src/igvfd/searches/configs/publication.py (hunk 4 above)
Comment (khzlin) [reply src/igvfd/searches/configs/publication.py:74]: fixed

File: src/igvfd/searches/configs/tissue.py (hunk 5 above)
Comment (khzlin) [reply src/igvfd/searches/configs/tissue.py:44]: fixed

File: src/igvfd/searches/configs/whole_organism.py (hunk 6 above)
Comment (khzlin) [reply src/igvfd/searches/configs/whole_organism.py:54]: fixed

File: src/igvfd/searches/configs/primary_cell.py (hunk 2 above)
Comment (khzlin) [reply src/igvfd/searches/configs/primary_cell.py:42]: fixed

File: src/igvfd/searches/configs/construct_library.py [hunk 13]
```diff
@@ -47,7 +72,9 @@ def cell_line():
             },
             'reporter_library_details': {
                 'title': 'Reporter Library Details'
-            }
+            },
+            'expression_library_details': {
```
Comment (iyoungworth) [inline src/igvfd/searches/configs/construct_library.py:76]: Just missing the word 'vector' here, should be "expression_vector_library_details"
Also adds a trailing comma

**Review by iyoungworth — CHANGES_REQUESTED** [review]
One last change in ConstructLibrary, otherwise I think good to go!

File: src/igvfd/searches/configs/construct_library.py (hunk 13 above)
Comment (khzlin) [reply src/igvfd/searches/configs/construct_library.py]: thanks for catching that. Should be fixed now.

**Review by iyoungworth — CHANGES_REQUESTED** [review]

**Review by iyoungworth — DISMISSED** [review]

File: src/igvfd/searches/configs/construct_library.py [hunk 14]
```diff
@@ -73,8 +73,8 @@ def construct_library():
             'reporter_library_details': {
                 'title': 'Reporter Library Details'
             },
-            'expression_library_details': {
-                'title': 'Expression Library Details'
+            'expression_vector_library_details': {
+                'title': 'Expression Vectory Library Details'
```
[the remark this answers is not in the document]
Comment (khzlin) [reply src/igvfd/searches/configs/construct_library.py]: fixed

File: src/igvfd/searches/configs/curated_set.py [hunk 15]
```diff
@@ -25,10 +22,16 @@ def curated_set():
             'curated_set_type': {
                 'title': 'Curated Set Type'
             },
+            'collections': {
```
Comment (iyoungworth) [inline src/igvfd/searches/configs/curated_set.py:25]: Collections are duplicated here

File: src/igvfd/searches/configs/rodent_donor.py [hunk 16]
```diff
@@ -38,6 +38,7 @@ def rodent_donor():
                 'facet_fields': [
                     'strain_background',
                     'sex',
+                    'virtual',
```
Comment (iyoungworth) [inline src/igvfd/searches/configs/rodent_donor.py:41]: Collection is missing an 's' in L47, not sure if I missed or is from another ticket

File: src/igvfd/searches/configs/signal_file.py [hunk 17]
```diff
@@ -22,6 +22,72 @@ def signal_file():
...
             },
+            'status': {
+                'title': 'Status'
+            },
+            'strand_specificity': {
+                'title': 'Strand Specificity'
+            },
+            'collections': {
+                'title': 'Collections'
+            },
```
Comment (iyoungworth) [inline src/igvfd/searches/configs/signal_file.py:33]: Your lab.title here is preferred, please remove duplicate lab in L22

File: src/igvfd/searches/configs/tissue.py [hunk 18]
```diff
@@ -48,8 +51,11 @@ def tissue():
                     'sample_terms.term_name',
                     'disease_terms.term_name',
                     'treatments.treatment_term_name',
+                    'biosample_term.term_name',
```
Comment (iyoungworth) [inline src/igvfd/searches/configs/tissue.py:54]: L54 biosample_term needs removal

File: src/igvfd/searches/configs/multiplexed_sample.py [hunk 19]
```diff
@@ -7,6 +7,15 @@
 def multiplexed_sample():
     return {
         'facets': {
+            'biosample_terms.term_name': {
```
Comment (iyoungworth) [inline src/igvfd/searches/configs/multiplexed_sample.py:10]: Should be 'sample_terms'

File: src/igvfd/searches/configs/multiplexed_sample.py [hunk 20]
```diff
@@ -21,8 +30,37 @@ def multiplexed_sample():
...
+            },
+            'biomarkers.classification': {
+                'title': 'Biomarkers Classification'
             }
         },
+        'facet_groups': [
+            {
+                'title': 'Sample',
+                'facet_fields': [
+                    'biosample_terms.term_name',
```
Comment (iyoungworth) [inline src/igvfd/searches/configs/multiplexed_sample.py:42]: should be 'sample_terms'

**Review by iyoungworth — CHANGES_REQUESTED** [review]
A few small corrections to make

File: src/igvfd/searches/configs/curated_set.py (hunk 15 above)
Comment (khzlin) [reply src/igvfd/searches/configs/curated_set.py:25]: done

File: src/igvfd/searches/configs/rodent_donor.py (hunk 16 above)
Comment (khzlin) [reply src/igvfd/searches/configs/rodent_donor.py:41]: added s

File: src/igvfd/searches/configs/signal_file.py (hunk 17 above)
Comment (khzlin) [reply src/igvfd/searches/configs/signal_file.py:33]: done

File: src/igvfd/searches/configs/tissue.py (hunk 18 above)
Comment (khzlin) [reply src/igvfd/searches/configs/tissue.py:54]: done

File: src/igvfd/searches/configs/multiplexed_sample.py (hunk 19 above)
Comment (khzlin) [reply src/igvfd/searches/configs/multiplexed_sample.py:10]: done

File: src/igvfd/searches/configs/multiplexed_sample.py (hunk 20 above)
Comment (khzlin) [reply src/igvfd/searches/configs/multiplexed_sample.py:42]: done

File: src/igvfd/searches/configs/signal_file.py [hunk 21]
```diff
@@ -6,20 +6,83 @@
...
+            'accession': {
+                'title': 'Accession'
+            },
+            'content_type': {
+                'title': 'Content Type'
+            },
+            'file_format': {
+                'title': 'File Format'
+            },
+            'status': {
```
Comment (iyoungworth) [inline src/igvfd/searches/configs/signal_file.py:22]: Status is duplicated here

File: src/igvfd/searches/configs/in_vitro_system.py [hunk 22]
```diff
@@ -42,6 +42,9 @@ def in_vitro_system():
             },
             'virtual': {
                 'title': 'Virtual'
+            },
+            'biomarkers.classification': {
+                'titlle': 'Biomarkers Classification'
```
[the remark this answers is not in the document]
Comment (khzlin) [reply src/igvfd/searches/configs/in_vitro_system.py]: i don't see it on my end

File: src/igvfd/searches/configs/signal_file.py (hunk 21 above)
Comment (khzlin) [reply src/igvfd/searches/configs/signal_file.py:22]: done

File: src/igvfd/searches/configs/signal_file.py [hunk 23]
```diff
@@ -6,20 +6,83 @@
...
+                'title': 'Alternate Accessions'
+            },
             'content_type': {
                 'title': 'Content Type'
             },
             'file_format': {
                 'title': 'File Format'
             },
-            'lab': {
+            'lab.title': {
```
[the remark this answers is not in the document]
Comment (khzlin) [reply src/igvfd/searches/configs/signal_file.py]: i don't see this on my end.

**Review by iyoungworth — APPROVED** [review]
Looks great! I think Github view got weird because I had an unsubmitted review, I didn't see any remaining issues

**Pull request merged**